test/specs/anInitialSuite.e2e.ts

```diff
@@ -77,7 +77,8 @@ describe('An Initial Suite', async () => {
           break;
       }
     }
-    expect(unexpectedSfdxCommandWasFound).toBe(false);
+    // TODO: add this back in once this bug has been fixed
```

Comment:
This expectation fails.  This will be aded back in once the bug has been fixed.

Reply:
Are you sure about this? I left you some comments in slack last week. This is supposed to be fixed since https://github.com/forcedotcom/salesforcedx-vscode/pull/4749

Reply:
Thanks @CristiCanizales !  I just took a look and you're right.  It's working again and I added this check back in.